prog/weavewait/main.go

```diff
@@ -26,6 +26,11 @@ func main() {
 		args = args[1:]
 	}
 
+	if len(args) > 0 && args[0] == "-c" { // tell pid 1 to continue
+		checkErr(syscall.Kill(1, syscall.SIGUSR2))
```

Comment:
I read the Docker container startup code at length; it is quite confusing as there is a lot of duplication between the 'regular start' case and the 'start a process inside an already-existing container' case.

However, it seems pretty clear that they expect the "main" process of the container to have PID 1, as they refer to it as the "init process".

Post #824, we don't attempt to signal in the other case, e.g. when a process is started by `docker exec`.  But this change makes that line of reasoning invalid, so perhaps we should try to do the signal.